lib/serializables/log-event.js

```diff
@@ -83,13 +77,11 @@ function LogEventFactory (
         var statement = [];
 
         this.context = LogEvent.redact(this.context);
-        statement.push('[%s]'.format(this.level));
+        // The level length 8 is hardcoded here, as the loggest level up to now
+        // is 8, this might be subject to change when new log levels are added
+        // in the future
+        statement.push('[%s]'.format( _.pad(this.level, 8)));
         statement.push('[%s]'.format(this.timestamp));
-        statement.push('[%s]'.format(this.name));
-        statement.push('[%s]'.format(this.module));
-        statement.push('[%s]'.format(
```

Comment:
we still need [Server], when having nodeId, it will display [<nodeId>] such as [8d2f33]